[2025-06-09T11:45:51.842+0200] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-06-09T11:45:51.853+0200] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: backfill_pollution_weather_dag.combine_backfill_data manual__2025-06-09T09:45:49.837307+00:00 [queued]>
[2025-06-09T11:45:51.856+0200] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: backfill_pollution_weather_dag.combine_backfill_data manual__2025-06-09T09:45:49.837307+00:00 [queued]>
[2025-06-09T11:45:51.856+0200] {taskinstance.py:2303} INFO - Starting attempt 1 of 2
[2025-06-09T11:45:51.876+0200] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): combine_backfill_data> on 2025-06-09 09:45:49.837307+00:00
[2025-06-09T11:45:51.878+0200] {standard_task_runner.py:63} INFO - Started process 12561 to run task
[2025-06-09T11:45:51.881+0200] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'backfill_pollution_weather_dag', 'combine_backfill_data', 'manual__2025-06-09T09:45:49.837307+00:00', '--job-id', '353', '--raw', '--subdir', 'DAGS_FOLDER/backfill_dag.py', '--cfg-path', '/tmp/tmpk9cvwy_z']
[2025-06-09T11:45:51.882+0200] {standard_task_runner.py:91} INFO - Job 353: Subtask combine_backfill_data
[2025-06-09T11:45:51.918+0200] {task_command.py:426} INFO - Running <TaskInstance: backfill_pollution_weather_dag.combine_backfill_data manual__2025-06-09T09:45:49.837307+00:00 [running]> on host DESKTOP-2QS8J6D.
[2025-06-09T11:45:51.988+0200] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='backfill_pollution_weather_dag' AIRFLOW_CTX_TASK_ID='combine_backfill_data' AIRFLOW_CTX_EXECUTION_DATE='2025-06-09T09:45:49.837307+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-06-09T09:45:49.837307+00:00'
[2025-06-09T11:45:51.989+0200] {taskinstance.py:430} INFO - ::endgroup::
[2025-06-09T11:45:56.385+0200] {logging_mixin.py:188} INFO - [COMBINING] City: istanbul
[2025-06-09T11:45:59.799+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine istanbul 2025-05-20: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:00.354+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine istanbul 2025-05-21: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:00.953+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine istanbul 2025-05-22: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:01.455+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine istanbul 2025-05-23: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:01.889+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine istanbul 2025-05-24: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:02.287+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine istanbul 2025-05-25: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:02.688+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine istanbul 2025-05-26: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:03.079+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine istanbul 2025-05-27: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:03.079+0200] {logging_mixin.py:188} INFO - [COMBINING] City: moscow
[2025-06-09T11:46:03.475+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine moscow 2025-05-20: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:03.868+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine moscow 2025-05-21: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:04.272+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine moscow 2025-05-22: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:05.615+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine moscow 2025-05-23: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:05.971+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine moscow 2025-05-24: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:06.324+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine moscow 2025-05-25: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:06.679+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine moscow 2025-05-26: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:07.019+0200] {logging_mixin.py:188} INFO - [ERROR] Failed to combine moscow 2025-05-27: [COLUMN_ALREADY_EXISTS] The column `lat` already exists. Choose another name or rename the existing column. SQLSTATE: 42711
[2025-06-09T11:46:07.960+0200] {logging_mixin.py:188} INFO - [DONE] Backfill combining completed
[2025-06-09T11:46:07.960+0200] {python.py:237} INFO - Done. Returned value was: None
[2025-06-09T11:46:07.960+0200] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-06-09T11:46:07.964+0200] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=backfill_pollution_weather_dag, task_id=combine_backfill_data, execution_date=20250609T094549, start_date=20250609T094551, end_date=20250609T094607
[2025-06-09T11:46:08.016+0200] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-06-09T11:46:08.020+0200] {taskinstance.py:3482} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-06-09T11:46:08.020+0200] {local_task_job_runner.py:222} INFO - ::endgroup::
